substrate/bin/node-template/runtime/src/lib.rs

```diff
@@ -105,7 +105,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	//   `spec_version`, and `authoring_version` are the same between Wasm and native.
 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
 	//   the compatible custom types.
-	spec_version: 100,
+	spec_version: 1,
```

Comment:
the comment says it should be 100. will this also be an issue for other runtime that you set version to 1?

Reply:
FML